tests/article.test.js

```diff
@@ -0,0 +1,193 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+import user from './index.test';
+import models from '../models';
+import dummy from './dummy';
+
+chai.use(chaiHttp);
+
+const { users } = models;
+
+let authToken;
+let articleSlug;
+
+describe('CRUD articles Routes', () => {
+  before(() => {
```

Comment:
To further help you on the matter why seeders are important, in this case, you have to run two blocks of code in order to carry out the main test. You could have used seeders so that you just login the seeded user